addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-mount-service.js

```diff
@@ -32,8 +33,8 @@ const readWritePermissionLevel = 'readwrite';
 const readWriteStatementId = 'S3StudyReadWriteAccess';
 const readOnlyStatementId = 'S3StudyReadAccess';
 
-const parseS3Arn = arn => {
-  const path = arn.slice('arn:aws:s3:::'.length);
+const parseS3Arn = (arn, partition) => {
```

Comment:
`arn` should already contain partition

I believe AWS SDK has better API to parse arn string to structure data